gordo_components/machine/dataset/data_provider/azure_utils.py

```diff
@@ -29,12 +30,16 @@ def get_datalake_token(
     lib.DataLakeCredential
         A lib.DataLakeCredential which can be used to authenticate towards the datalake
     """
-
+    dl_service_auth_str = (
+        os.environ.get("DL_SERVICE_AUTH_STR")
+        if str(dl_service_auth_str) == "None"
+        else dl_service_auth_str
```

Comment:
Is the string casting necessary? Coulden't we just check if it is None?

Reply:
Ya, at some point I got literally `"None"` and thought this would cover both aspects ok enough.. probably should make an issue later to sort it out?

Reply:
Sounds a bit funny getting "None" :laughing:, so perhaps we should look into that, however this should cover both cases as you say :+1: